apps/console/src/extensions/components/groups/wizard/create-group-wizard.tsx

```diff
@@ -80,7 +83,6 @@ interface WizardStepInterface {
 /**
  * Component to handle addition of a new group to the system.
  *
- * @param props props related to the create group wizard
```

Comment:
Hope this is intentional :)

Reply:
Fixed with aa6b04cb1d9612a46456994d7aeccf49e7b02602